### DISCUSSION TITLE
請教自建伺服器 ，python turtle 類的題目如何配置?

### DISCUSSION THREAD

**Discussion by dailinyang** [hydro-dev/Hydro#784] [Q&A]

**Comment by undefined-moe** [comment]
使用 `nix build github:hydro-dev/nix-channel#judge` 构建完整的 rootfs，将其（含 `/nix` 目录）映射至沙箱目录下，题目配置遵照官示例。

**Reply by SiliconXu** [reply]
已经运行 nix build，并修改了 mount.yaml（按照官方文档中的“编译”说明）。我检查了官方示例，下载整个题目并导入到自己的 OJ 中，但是在编辑器中无法看到 py3graphics 这个语言。这个是否需要自己创建？

**Comment by Tivonfeng** [comment]
请问您python turtle的题目配置成功了吗？求教

**Comment by dailinyang** [comment]
沒有

Tivonfeng ***@***.***>於 2024年1月23日 週二，下午6:59寫道：
